x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/optimizer/FoldNull.java

```diff
@@ -8,9 +8,8 @@
 package org.elasticsearch.xpack.esql.optimizer;
 
 import org.elasticsearch.xpack.ql.expression.Expression;
-import org.elasticsearch.xpack.ql.optimizer.OptimizerRules;
 
-public class FoldNull extends OptimizerRules.FoldNull {
+public class FoldNull extends LogicalPlanOptimizer.FoldNull {
```

Comment:
I don't think this class is needed anymore.

Reply:
We still need it unfortunately, It's there because FoldNull.rule() is protected, and it's used also by `AbstractFuncitonTestCase`

Reply:
Unless we want to make it public in ESQL